test/Conversion/ExportVerilog/instance-choice.mlir

```diff
@@ -0,0 +1,40 @@
+// RUN: circt-opt %s -export-split-verilog='dir-name=%t.dir'
+// RUN: cat %t.dir/top.sv | FileCheck %s
+
+hw.module private @TargetA(in %a: i32, out b: i32) {
+  hw.output %a : i32
+}
+
+hw.module private @TargetB(in %a: i32, out b: i32) {
+  hw.output %a : i32
+}
+
+hw.module private @TargetDefault(in %a: i32, out b: i32) {
+  hw.output %a : i32
+}
+
+// CHECK-LABEL: module top
+hw.module public @top(in %a: i32, out b: i32, out d: i32) {
+  // CHECK:      `ifndef __circt_choice_top_inst1
+  // CHECK-NEXT: `define __circt_choice_top_inst1 TargetDefault
+  // CHECK-NEXT: `endif
+  // CHECK-NEXT: __circt_choice_top_inst1 inst1 (
```

Comment:
Nit: shouldn't this be: 

```suggestion
  // CHECK-NEXT: `__circt_choice_top_inst1 inst1 (
```

Ditto below.

Reply:
The code is already generating the `` ` ``, so everything should the check is just not strict enough!